src/github.com/getlantern/flashlight/logging/error_aggregator.go

```diff
@@ -0,0 +1,176 @@
+package logging
```

Comment:
You are more aware of the types of errors there are. Do you think aggregating some classes is worth it? Can we identify them and work with those?